llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -804,14 +804,23 @@ class StoreInst final : public Instruction {
   }
 
 public:
+  /// Return true if this is a load from a volatile memory location.
+  bool isVolatile() const { return cast<llvm::LoadInst>(Val)->isVolatile(); }
```

Comment:
The build fails because this should be: `cast<llvm::StoreInst>(Val)`, not `llvm::LoadInst`.

Reply:
Yeah I see the issue, this didn't get committed properly. I fixed it.